crucible-client-types/src/lib.rs

```diff
@@ -139,3 +139,12 @@ pub enum ReplacementRequestCheck {
     /// original.
     ReplacementMatchesOriginal,
 }
+
+/// Extent information about a region.
+#[derive(Serialize, Deserialize, Clone, Debug, JsonSchema)]
+#[serde(tag = "type", rename_all = "snake_case")]
+pub struct RegionExtentInfo {
```

Comment:
Nit: docstrings would be nice here (especially for `block_size`, which is presumably in bytes?)

Reply:
Will do.  This was just moved but I can improve the comment(s) on it.